hackathon_site/event/views.py

```diff
@@ -1,9 +1,14 @@
+from django.views.generic import TemplateView
 from rest_framework import generics, mixins
 
 from event.models import User
 from event.serializers import UserSerializer
 
 
+class IndexView(TemplateView):
```

Comment:
Don't you need a get or post method here

Reply:
Nope - that's what `TemplateView` does: https://github.com/django/django/blob/master/django/views/generic/base.py#L157